### PR TITLE
test/integ/SAM debug: also test Image Lambdas

### PR DESCRIPTION
**Pull request by rli**
## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by hunterwerlla** [comment]
Can we also edit/remove the existing static files that point to beta images?

**Comment by rli** [comment]
> Can we also edit/remove the existing static files that point to beta images?

good call, fixed

File: src/integrationTest/sam.test.ts [hunk 1]
```diff
@@ -365,14 +465,18 @@ describe('SAM Integration Tests', async function () {
...
                 name: samApplicationName,
                 location: location,
-                template: helloWorldTemplate,
-                runtime: scenario.runtime,
                 dependencyManager: getDependencyManager(scenario.runtime),
             }
+            if (scenario.baseImage) {
+                initArguments.baseImage = scenario.baseImage
+            } else {
+                initArguments.runtime = scenario.runtime
```
Comment (bryceitoc9) [inline src/integrationTest/sam.test.ts:477]: Does this contradict the comment here? https://github.com/aws/aws-toolkit-vscode/pull/1423/files#diff-4b86e3cc81505da2a3b4c8faf1f8ac6d616ae2dec5488e8656d083bc111a1f94R386

File: src/testFixtures/workspaceFolder/python3.7-image-sam-app/hello_world/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM 628053151772.dkr.ecr.sa-east-1.amazonaws.com/awslambda/python3.7-runtime:beta
+FROM public.ecr.aws/lambda/python:3.7
```
Comment (bryceitoc9) [inline src/testFixtures/workspaceFolder/python3.7-image-sam-app/hello_world/Dockerfile:1]: (comment not specific to this file)

Do we need to cover the other runtimes? Why are only three represented in this PR?

File: src/integrationTest/sam.test.ts [hunk 3]
```diff
@@ -262,7 +352,7 @@ describe('SAM Integration Tests', async function () {
                 })
 
                 it('invokes and attaches on debug request (F5)', async function () {
-                    setTestTimeout(this.test?.fullTitle(), 60000)
+                    setTestTimeout(this.test?.fullTitle(), 90000)
```
Comment (bryceitoc9) [inline src/integrationTest/sam.test.ts:355]: Any specific reasons for the time bump?

Comment (justinmk3) [reply src/integrationTest/sam.test.ts:355]: If download takes a long time then yes we need to increase test timeout.

**Review by justinmk3 — APPROVED** [review]

**Pull request merged**